subsys/net/lib/nrf_cloud/coap/src/nrf_cloud_coap_transport.c

```diff
@@ -642,10 +643,57 @@ int nrf_cloud_coap_transport_init(struct nrf_cloud_coap_client *const client)
 	return err;
 }
 
+static int nrf_cloud_coap_try_address(struct sockaddr *const addr)
+{
+	int err;
+	int sock;
+	size_t addr_size;
+
+	LOG_DBG("Creating socket type IPPROTO_DTLS_1_2");
+	sock = socket(addr->sa_family, SOCK_DGRAM, IPPROTO_DTLS_1_2);
+
+	if (sock < 0) {
+		LOG_INF("Failed to create CoAP socket, errno: %d", errno);
```

Comment:
nit: i know these are existing logs, but i would prefer these INFs that print "failed" be DBG messages if they do not indicate that something is wrong or they are a normal occurrence.
or at least consolidate into one log message under the `out:` label, printing err.

Reply:
Good point, I'll either change this to DBG or I'll change the wording